game_eggs/minecraft/java/neoforge/egg-neo-forge.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": "https://mirror.invalidpanda.dev/neoforge/egg-neo-forge.json"
+    },
+    "exported_at": "2024-02-14T21:28:58+01:00",
+    "name": "NeoForge",
+    "author": "mail@invalidpanda.dev",
+    "description": "A new fork of the Minecraft modding platform Forge. NeoForge provides support for many Forge mods, but some newer mods are NeoForge-exclusive. NeoForge supports Minecraft version 1.20.1 and newer.",
+    "features": null,
```

Comment:
The ulau feature is missing?

Reply:
Oh sure, I'll add it! Weird that it worked anyways without it

Reply:
it works, it will just not give the eula popup